DigitalLearningSolutions.Data/Helpers/LearningHubSsoSecurityHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace DigitalLearningSolutions.Data.Helpers
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+    using DigitalLearningSolutions.Data.Services;
+
+    public class LearningHubSsoSecurityHelper

```

Comment:
To fully utilise dependency injection this will need an interface so this helper can be injected/mocked also.